src/mbgl/shader/circle_uniforms.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <mbgl/shader/uniforms.hpp>
+
+namespace mbgl {
+
+MBGL_DEFINE_UNIFORM_SCALAR(float, u_radius);
```

Comment:
Yeah, I've seen that; I'm referring to the situation when two shaders use the same name for different uniform types.